src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import { useState } from 'react';
+
+const ChatEntry = ({sender,body,timeStamp,updateLikes}) => {
+  let senderClass = 'chat-entry local';
+  if(sender === 'Estragon') {
+    senderClass = 'chat-entry remote';
+  }
+  const [like, numLike] = useState(false);
 
-const ChatEntry = (props) => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={senderClass}>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time">
+          <TimeStamp time={timeStamp}/>
+          </p>
+        <button className="like" onClick={()=> updateLikes(like,numLike)}>
+          {like ? '❤️': '🤍' }
```

Comment:
The ternary is a great tool for situations like this, happy to see you all implementing it!